packages/core/src/browser.js

```diff
@@ -124,68 +166,37 @@ export default class Browser extends EventEmitter {
       }));
     }
 
-    // trigger rejecting pending page callbacks
-    for (let page of this.pages.values()) {
-      page._handleClose();
+    // trigger rejecting pending session callbacks
+    for (let session of this.sessions.values()) {
+      session._handleClose();
     }
 
-    // clear callback and page references
+    // clear own callbacks and sessions
```

Comment:
? nitpick
```suggestion
    // clear our own callbacks and sessions
```

Reply:
Nitpick to your nitpick

```suggestion
    // clear this instance's own callbacks and sessions
```

FWIW, "\<verb> \<pronoun>" (without a proper noun) is grammatically correct. 😜

Reply:
For that, I’m rejecting the PR